### PR TITLE
Adding Mouse panning to the PDA

### PR DESCRIPTION
**Pull request by ifeldshteyn**
Add mouse panning to the PDA

### CODE REVIEW

File: TheForceEngine/TFE_DarkForces/GameUI/pda.cpp [hunk 1]
```diff
@@ -350,6 +352,32 @@ namespace TFE_DarkForces
 		// Ensure that mouse movements are not lost between updates.
 		pda_handleMouseAccum(&wdx, &wdy);
 
+		// Handle Mouse Panning
+		if (TFE_Input::mouseDown(MBUTTON_LEFT))
+		{
+			s32 mouseX, mouseZ;
+			SDL_GetMouseState(&mouseX, &mouseZ);
```
Comment (luciusDXL) [inline TheForceEngine/TFE_DarkForces/GameUI/pda.cpp:359]: TFE_Input has a function to get the mouse position, so you don't need to include SDL here.

**Review by luciusDXL — CHANGES_REQUESTED** [review]

File: TheForceEngine/TFE_DarkForces/GameUI/pda.cpp (hunk 1 above)
Comment (ifeldshteyn) [reply TheForceEngine/TFE_DarkForces/GameUI/pda.cpp]: This is implemented.

File: TheForceEngine/TFE_DarkForces/GameUI/pda.cpp [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #include <cstring>
-
+#include <SDL.h>
```
Comment (luciusDXL) [inline TheForceEngine/TFE_DarkForces/GameUI/pda.cpp:2]: You should no longer need this include.

**Review by luciusDXL — CHANGES_REQUESTED** [review]

File: TheForceEngine/TFE_DarkForces/GameUI/pda.cpp (hunk 2 above)
Comment (ifeldshteyn) [reply TheForceEngine/TFE_DarkForces/GameUI/pda.cpp]: SDL is now removed in [bda14bc](https://github.com/luciusDXL/TheForceEngine/pull/459/commits/bda14bc28b486a53c034c4c8f425474dc863a6d1)

**Review by luciusDXL — APPROVED** [review]

**Pull request merged**